Foundation/NSGeometry.swift

```diff
@@ -884,22 +882,20 @@ extension NSCoder {
 
 private extension NSCoder {
     func _encodeCGFloat(_ value: CGFloat) {
-        if let keyedArchiver = self as? NSKeyedArchiver {
-            keyedArchiver._encodeValue(NSNumber(value: value.native))
-        } else {
-            NSUnimplemented()
+        guard let keyedArchiver = self as? NSKeyedArchiver else {
+            preconditionFailure("Unkeyed coding is unsupported.")
```

Comment:
Putting aside this oddity of the abstract protocol knowing about NSKeyedArchiver for a moment, instead of letting NSKeyedArchiver implement this correctly -- we've decided across the project to just not support unkeyed coding instead of haphazardly implementing in some places and not others.